app/helpers/Intersect.js

```diff
@@ -101,14 +102,62 @@ export async function IntersectSet(concernLocationArray, completion) {
   });
 }
 
+// Function to determine if two location points are "nearby".
```

Comment:
use jsdoc:

```
/**
 * 
 */
```

That way many IDEs will show the method description on hover of the method anywhere in the code base. (IDEA and VScode)